drivers/syslog/vsyslog.c

```diff
@@ -71,6 +71,11 @@ int nx_vsyslog(int priority, FAR const IPTR char *fmt, FAR va_list *ap)
 {
   struct lib_syslogstream_s stream;
   int ret;
+#if defined(CONFIG_SYSLOG_TIMESTAMP_FORMATED)
+  time_t time;
+  struct tm * tm;
```

Comment:
remove *, tm need pass to localtime_r gmtime_r

Reply:
Yep, sorry. Corrected. Tested, and builds correctly in all configurations.